README.md

```diff
@@ -41,6 +41,20 @@ tmpl.render({"what", "World"}, [&ss](const std::string& str) {
 // ss.str() == "Hello World!"
 ````
 
+### Usage with cmake
+
+```cmake
+find_package(mustache REQUIRED)
+
+add_executable(my_main src/main.cpp)
+target_link_libraries(${PROJECT_NAME} mustache::mustache)
```

Comment:
can we simplify this by removing `add_executable` and just use a dummy target like "my_target" instead of PROJECT_NAME? because that is more obvious for an example (and project name is not always correct, so I'd rather use dummy examples).